torch/csrc/jit/passes/alias_analysis.h

```diff
@@ -44,17 +41,22 @@ class AliasDb {
   bool hasUntrackedEffects(Node* n) const;
 
   // Get all the values that `n` writes to.
-  std::unordered_set<const Value*> getWrites(Node* n) const;
+  // NOTE: this only returns values directly written to, not aliases thereof
+  //
+  // if `recurseBlocks` is true, gather writes on the nodes in `n`s sub-blocks
+  std::unordered_set<const Value*> getWrites(
```

Comment:
nit: a type alias for a set of values would be a good idea because we may change the implementation for speed.
For instance, `std::vector<bool>` keyed by a Value's unique id is potentially much smaller and faster than this current implementation, and we shouldn't make it really hard for that to be possible.